.github/workflows/release.yml

```diff
@@ -17,6 +17,20 @@ jobs:
         with:
           github-token: ${{ secrets.CHANGELOG_RELEASE }}
           version-file: "./package.json,./package-lock.json"
+          git-branch: "release-from-${{ github.sha }}"
+
+      - name: create PR with release info
+        id: create-pr
+        run: gh pr create -B main -H release-from-${{ github.sha }} --title 'Merge new release into main' --body 'Created by Github action'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we need to state this explicitly? GitHub Actions should have access to GitHub CLI without us explicitly giving it a token, but I've had to do it this way before even with that.

Reply:
I don't know. I think at least for the part where we merge the new branch, we'll need a token with elevated permissions. I will spin up a test-repository and try the action, will link the results here.

Reply:
I did some testing and all the actions do in fact only work when the token is provided explicitly.

Reply:
Bah! Same problem that happened to me recently. The docs are a lie 😂😅

Reply:
> Bah! Same problem that happened to me recently. The docs are a lie 😂😅

Kinda off-topic: you can submit an issue/PR in the [`docs`](https://github.com/github/docs) repository about it. 🙂